net/tn5250/Portfile

```diff
@@ -16,11 +17,9 @@ long_description    tn5250 is a telnet client for the IBM iSeries and AS/400 \
                     that emulates 5250 terminals and printers.
 
 homepage            https://tn5250.github.io/
-
-checksums           rmd160  5b788fe4bd46ec446d30924bdbb6c89cace73e88 \
-                    sha256  f6383bc62360d858d55337296f373fb1797a2ba39dc931cd09d64a4a558e1641 \
-                    size    700576
-
+checksums           rmd160  466d0f7f2eb05c6d5ad89c033732e794ddfe4a2e \
+                    sha256  b936d39d5210fe447245cf08778655a48876ae2dfcb67373910163602128a919 \
+                    size    645380
 
 configure.args-append      --with-ssl
```

Comment:
this should likely become `--with-ssl=[openssl::install_area]`

Reply:
nevermind, it seems like the configure script figures this out correctly. Thanks!